modules/yieldlabBidAdapter.js

```diff
@@ -39,6 +39,9 @@ export const spec = {
       if (bid.params.targeting) {
         query.t = createQueryString(bid.params.targeting)
       }
+      if (bid.userIdAsEids) {
```

Comment:
Because your function `createUserIdString` loops over the length of the array I would recommend having this check be `Array.isArray(bid.userIdAsEids` Just in case something weird happens where `userIdAsEids` is on the bidRequest object but is not an array (undefined, object, int, etc..)

Reply:
Hey @robertrmartinez! Thanks for taking a look at this. I added your proposed changes. Makes sense!